config/crd/bases/controlplane.cluster.x-k8s.io_k0scontrolplanetemplates.yaml

```diff
@@ -0,0 +1,225 @@
+                                - proxy
+                                type: string
+                              serverAddress:
+                                description: Server address of the tunneling server.
+                                  If empty, k0smotron will try to detect worker node
+                                  address for.
+                                type: string
+                              serverNodePort:
+                                default: 31700
+                                description: NodePort to publish for server port of
+                                  the tunneling server. If empty, k0smotron will use
+                                  the default one.
+                                format: int32
+                                type: integer
+                              tunnelingNodePort:
+                                default: 31443
+                                description: NodePort to publish for tunneling port.
+                                  If empty, k0smotron will use the default one.
+                                format: int32
+                                type: integer
+                            type: object
+                        type: object
+                      k0sVersion:
+                        type: string
+                      machineTemplate:
+                        properties:
+                          infrastructureRef:
+                            description: InfrastructureRef is a required reference
+                              to a custom resource offered by an infrastructure provider.
+                            properties:
+                              apiVersion:
+                                description: API version of the referent.
+                                type: string
+                              fieldPath:
+                                description: 'If referring to a piece of an object
+                                  instead of an entire object, this string should
+                                  contain a valid JSON/Go field access statement,
+                                  such as desiredState.manifest.containers[2]. For
+                                  example, if the object reference is to a container
+                                  within a pod, this would take on a value like: "spec.containers{name}"
+                                  (where "name" refers to the name of the container
+                                  that triggered the event) or if no container name
+                                  is specified "spec.containers[2]" (container with
+                                  index 2 in this pod). This syntax is chosen only
+                                  to have some well-defined way of referencing a part
+                                  of an object. TODO: this design is not final and
+                                  this field is subject to change in the future.'
```

Comment:
This string becomes:

```console
If referring to a piece of an object
                                  instead of an entire object, this string should
                                  contain a valid JSON/Go field access statement,
                                  such as desiredState.manifest.containers[2]. For
                                  example, if the object reference is to a container
                                  within a pod, this would take on a value like: "spec.containers{name}"
                                  (where "name" refers to the name of the container
                                  that triggered the event) or if no container name
                                  is specified "spec.containers[2]" (container with
                                  index 2 in this pod). This syntax is chosen only
                                  to have some well-defined way of referencing a part
                                  of an object. TODO: this design is not final and
                                  this field is subject to change in the future.
```